src/slurm_plugin/slurm_resources.py

```diff
@@ -414,7 +427,31 @@ def is_backing_instance_valid(self, log_warn_if_unhealthy=True):
                         self,
                         self.state_string,
                     )
-                return False
+                # Allow a few iterations for the eventual consistency of EC2 data
+                logger.info(
+                    f"ec2_backing_instance_max_count {ec2_backing_instance_max_count} "
+                    f"nodes_without_backing_instance_count_map {nodes_without_backing_instance_count_map}"
+                )
+                if any(
+                    args in [None]
+                    for args in [ec2_backing_instance_max_count, nodes_without_backing_instance_count_map]
+                ):
+                    logger.info("No max count or map provided, ignoring backing instance timeout")
+                    return False
```

Comment:
I think this is useless since the ec2_backing_instance_max_count has a default value. Even if it's not present in the config, its value is always set

Reply:
The default value is None though.